### PR TITLE
Spring Data JPA

### PR DESCRIPTION
**Pull request by TemkaHillfinger**

### CODE REVIEW

File: src/main/java/bookstore/dto/BookDto.java [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
+import jakarta.persistence.Column;
 import java.math.BigDecimal;
+import lombok.Data;
 
+@Data
 public class BookDto {
     private Long id;
     private String title;
+    @Column(name = "author")
```
Comment (olekskov) [inline src/main/java/bookstore/dto/BookDto.java:11]: why column annotation in DTO?

File: src/main/java/bookstore/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,9 +1,13 @@
 package bookstore.dto;
 
+import jakarta.persistence.Column;
 import java.math.BigDecimal;
+import lombok.Data;
 
+@Data
 public class CreateBookRequestDto {
     private String title;
+    @Column(name = "author")
```
Comment (olekskov) [inline src/main/java/bookstore/dto/CreateBookRequestDto.java:10]: same question

**Review by olekskov — CHANGES_REQUESTED** [review]
1. Task requirement "Implement soft delete concept for the Book entity" is not implemented
2. PUT: /api/books/{id} (Update a specific book) is not implemented
3. Some comments below

File: src/main/java/bookstore/model/Book.java [hunk 3]
```diff
@@ -5,12 +5,18 @@
...
+import org.hibernate.annotations.Where;
 
-@Entity
-@jakarta.persistence.Table(name = "books")
 @Data
+@Entity
+@SQLDelete(sql = "UPDATE books SET is_delete = true WHERE id = ?")
+@Where(clause = "is_deleted=false")
+@Table(name = "books")
+
```
Comment (Vasul-Pankiv) [inline src/main/java/bookstore/model/Book.java:19]: empty line

File: src/main/java/bookstore/model/Book.java [hunk 4]
```diff
@@ -25,4 +31,6 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
```
Comment (Vasul-Pankiv) [inline src/main/java/bookstore/model/Book.java:33]: in db comn has another name, Please add @Column annotation with necessary param

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-books-table.yaml
```
Comment (Vasul-Pankiv) [inline src/main/resources/db/changelog/db.changelog-master.yaml:3]: add empty line to the end of file

**Review by Vasul-Pankiv — CHANGES_REQUESTED** [review]

**Review by kozhukhovsky — APPROVED** [review]

**Pull request merged**